src/Modal/ModalContent.jsx

```diff
@@ -10,9 +10,19 @@ import style from './Modal.style'
 
 import { Box } from '@ivoryio/kogaio'
 
+const inIframe = () => {
+  try {
+    return window.self !== window.top
+  } catch (e) {
+    return true
+  }
```

Comment:
@sergiuelisei is there a scenario where this fails? `window` not being available maybe?

Reply:
@sergiuelisei also is it correct that we default to `true` if the check fails?